src/providers/TransactionBuilderContextProvider.tsx

```diff
@@ -732,6 +738,35 @@ const TransactionBuilderContextProvider = ({
 
   const [showMulticallOptions, setShowMulticallOptions] = useState<string | null>(null);
 
+  const addTransactionBlock = (
+    availableTransactionBlock: ITransactionBlock,
+    isBridgeTransactionBlockAndDisabled = false,
+    isKlimaBlockIncluded = false
+  ) => {
+    if (!availableTransactionBlock) return;
+
+    if (isKlimaBlockIncluded && transactionBlocks.length > 0) {
+      showAlertModal(
+        'Cannot add klima staking block with transaction batch. Please remove previous transactions or continue after the previous transactions are executed'
+      );
+      return;
+    }
+    if (hasKlimaBlockAdded) {
+      showAlertModal(
+        'Cannot add another transaction block with transaction batch. Please remove Klima transaction or continue after the klima transaction is executed'
+      );
+      return;
+    }
+    if (availableTransactionBlock.type === TRANSACTION_BLOCK_TYPE.DISABLED || isBridgeTransactionBlockAndDisabled)
+      return;
```

Comment:
sir why line 746 has return in one line and this breaks? just trying to understand the AI on your ide auto linter lol